plugin/pkg/scheduler/core/generic_scheduler.go

```diff
@@ -159,6 +168,64 @@ func (g *genericScheduler) selectHost(priorityList schedulerapi.HostPriorityList
 	return priorityList[ix].Host, nil
 }
 
+// preempt finds nodes with pods that can be preempted to make room for "pod" to
+// schedule. It chooses one of the nodes and preempts the pods on the node and
+// returns the node and the list of preempted pods if such a node is found.
+// TODO(bsalamat): Add priority-based scheduling. More info: today if many pods
+// with lower priority and smaller than the pending pod exist in the system, they
```

Comment:
You can fix this in a followup PR, but "exist in the system" is a little vague (and I don't think it actually needs to be "many" pods, unless I'm misunderstanding). I would change this sentence to "today one or more pending pods (different from the pod that triggered the preemption(s)) may schedule into some portion of the resources freed up by the preemption(s) before the pod that triggered the preemption(s) has a chance to schedule there, thereby preventing the pod that triggered the preemption(s) from scheduling."

Reply:
I updated it in this PR.